avocado/utils/memory.py

```diff
@@ -320,3 +320,26 @@ def get_thp_value(feature):
         return (re.search(r"\[(\w+)\]", value)).group(1)
     else:
         return value
+
+
+def memtotal_sys():
+    """
+    To get the total memory(ram) as detected by system
+    MemTotal in /proc/meminfo would display
+    total usable memory(i.e. physical ram minus
+    a few reserved bits and the kernel binary code)
```

Comment:
I'm a bit confused from this docstring. How about: `Reports actual memory size according to online-memory blocks available via "/sys"`?

Also please move this function next to `memtotal` to make it easier for people to find it.